integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6093,6 +6095,35 @@ msgstr "Diese Veranstaltung löschen"
 msgid "Date and time"
 msgstr "Datum und Uhrzeit"
 
+#: cms/templates/events/event_form_sidebar/date_and_time_box.html
+msgid ""
+"One-time events take place on a specific date. For recurring events (e.g. "
+"weekly) this can be set under “Recurring”."
+msgstr ""
+"Einmalige Veranstaltungen finden an einem bestimmten Termin statt. Bei "
+"wiederkehrenden Veranstaltungen (z.B. wöchentlich) kann dies unter "
+"“wiederkehrend” eingestellt werden."
+
+#: cms/templates/events/event_form_sidebar/date_and_time_box.html
+msgid ""
+"These events take place over a longer period of time. E.g. Christmas markets "
+"or similar."
+msgstr ""
+"Diese Veranstaltungen finden über einen längeren Zeitraum statt. Z.B. "
+"Weihnachtsmärkte o.ä."
+
+#: cms/templates/events/event_form_sidebar/date_and_time_box.html
+msgid "These events take place daily."
+msgstr "Diese Veranstaltungen finden täglich statt."
+
+#: cms/templates/events/event_form_sidebar/date_and_time_box.html
+msgid "Please create daily repeating events under “Long-term”."
+msgstr "Täglich wiederholende Events bitte unter “Fortlaufend” anlegen."
+
+#: cms/templates/events/event_form_sidebar/date_and_time_box.html
+msgid " (optional)"
+msgstr " (optional)"
```

Comment:
I think if possible we should try to avoid to translate empty spaces. Do you think it would be possible to exclude the empty space from the translation? :)

Reply:
Nice idea :) applied!